src/pages/settings/settings.html

```diff
@@ -32,17 +32,14 @@
         </ion-icon>
         <ion-label>
           <div *ngIf="walletsGroup[0].canSign">{{profileProvider.getWalletGroup(walletsGroup[0].keyId).name}}</div>
-          <div *ngIf="!walletsGroup[0].canSign" translate>Read Only Wallets</div>
+          <div *ngIf="!walletsGroup[0].canSign" translate>Read Only Key</div>
```

Comment:
Are we sure to rename Read Only wallets to Read only keys? Even if that kind of wallets doesn't have private key?